pkg/volume/vsphere_volume/attacher.go

```diff
@@ -81,7 +81,7 @@ func (attacher *vsphereVMDKAttacher) Attach(spec *volume.Spec, nodeName types.No
 		return "", err
 	}
 
-	return path.Join(diskByIDPath, diskSCSIPrefix+diskUUID), nil
+	return filepath.Join(diskByIDPath, diskSCSIPrefix+diskUUID), nil
```

Comment:
@gnufied brought up a good scenario.  What happens if the attach controller runs on a Windows master, but nodes are Linux?  I'm not sure if even the current code (using path) is working correctly, so there may be a bigger issue here.

I think for this PR, let's not change any path calls for Attach() functions, which are executed by master.

Reply:
As far as I know, master couldn't be run on Windows yet. It's possible soon by LCOW, but it's still linux containers.

Reply:
Alright, should I change it back or not - have we reached consensus on this?
@msau42 @feiskyer

Reply:
I think to be safe, let's not change any Attach() calls right now, although I think the existing `path` call also is broken in that scenario.  We'll have to revisit this when there is Windows master support.